roles/eda/templates/eda-activation-worker.deployment.yaml.j2

```diff
@@ -66,6 +66,54 @@ spec:
 {% endif %}
 {% if bundle_ca_crt  %}
       initContainers:
+      - name: wait-for-migrations
+        image: {{ _image }}
+        imagePullPolicy: '{{ image_pull_policy }}'
+        command:
+          - /bin/bash
```

Comment:
you can run `/bin/bash /usr/bin/wait-for-migrations` so you don't need to run chmod +x

Reply:
and/or, you can mount the file in a place where you can have write perms

Reply:
I tried those suggestions and a couple other ideas, no luck.  But baking in the script into the image worked.

Reply:
I don't have any OpenShift but I think we can `defaultMode` for the configmap for the `volumes` to change perms for the script file initially.

```yaml
  volumes:
    ...
    - name: wait-for-migrations-script
      configMap:
        name: wait-for-migrations-script-configmap
        defaultMode: 0500
        items:
          - key: wait-for-migrations
            path: wait-for-migrations
```

Refer to: https://access.redhat.com/solutions/7024882

Reply:
Oh! Thanks kurokobo!  I will make use of that pattern in the future for sure.  After talking with @Alex-Izquierdo , I think we are going to go with the management command approach implemented here:
* https://github.com/ansible/eda-server/pull/927/files

It will be a nice way to have the "script" available in all installation methods for EDA.

Reply:
@rooftopcellist the PR is merged.